frontend/apps/web/main.tsx

```diff
@@ -1,10 +1,11 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './src/App.tsx'
-import './src/index.css'
+import React from "react";
+import ReactDOM from "react-dom/client";
+import TestApp from "./src/TestApp.tsx";
+import "./src/index.css";
 
-ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
+ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
   <React.StrictMode>
-    <App showForms={true} />
+    {/* <App showForms={true} /> */}
```

Comment:
Here I've commented out the actual `App` to avoid adding more logic to that file. Although it could've been done too.